agentextension/build.gradle.kts

```diff
@@ -40,6 +40,8 @@ tasks {
       include("LICENSE")
       include("NOTICE")
       include("licenses/**")
+
+      // TODO : ignore classes that must remain in the agent as we do in 'relocateJavaagentLibs'
```

Comment:
[for reviewer] follow-up item to create as the extension currently includes classes from API/SDK through transitive dependencies.

Reply:
Actually after double-checking this was due to my previous attempt to ignore at file-level, only the semconv attributes are duplicated and that's very probably not a major issue. So I'll remove this comment to prevent any confusion.